### DISCUSSION TITLE
chia wallet

### DISCUSSION THREAD

**Discussion by mad0me** [Chia-Network/chia-blockchain#6704] [Support]
no balance in the chia GUI but in chiaexplorer.com i can see that there is the amount that i got from hpool

**Comment by loppefaaret** [comment]
ensure that not only the node is synced, but also the wallet is synced - they are syncing independant of each other

**Comment by BrandtH22** [comment]
@mad0me , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
